FND/DatesAndTimes/FinancialDates.rdf

```diff
@@ -597,9 +621,17 @@ The recurrence start date can also be relative to the starting Date of the overa
 		<skos:definition>time interval that has duration 1 day and that meets a Thursday</skos:definition>
 	</owl:NamedIndividual>
 	
+	<owl:DatatypeProperty rdf:about="&fibo-fnd-dt-fd;hasAcquisitionDate">
+		<rdf:type rdf:resource="&owl;FunctionalProperty"/>
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasObservedDateTime"/>
+		<rdfs:label>has acquisition date</rdfs:label>
+		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;CombinedDateTime"/>
+		<skos:definition>links an asset or owner/controller/controllee to the date or date and time of purchase</skos:definition>
```

Comment:
too vague - is it control or ownership?
something can be acquired without being purchased.

Reply:
How would you see us change this?  The definition explicitly says date and time of purchase.